internal/current_target.go

```diff
@@ -0,0 +1,58 @@
+package internal
+
+import (
+	"bufio"
+	"errors"
+	"fmt"
+	"os"
+	"strconv"
+	"strings"
+)
+
+type CurrentTarget struct {
+	Name    string
+	Version int
+}
+
+func LoadCurrentTarget(currentTargeFile string) (CurrentTarget, error) {
+	cur := CurrentTarget{}
+
+	file, err := os.Open(currentTargeFile)
+	if err != nil {
+		return cur, fmt.Errorf("Unable to read current-target file: %w", err)
+	}
+
+	scanner := bufio.NewScanner(file)
```

Comment:
Could [this](https://github.com/go-ini/ini) potentially help?

I guess that 3PP module would allow smth like this:
```
file, err := ini.Load(currentTargeFile)
if err != nil {
    return cur, fmt.Errorf("Unable to read current-target file: %w", err)
}
cfg := file.GetSection("")
cur.Name = cfg.Key("TARGET_NAME").MustString("")
cur.Version = cfg.Key("CUSTOM_VERSION").MustUint("")
```

Reply:
it might. actually sourceable shell script content, but an ini-parser might work. We just need to decide if its worth vetting that lib.

Reply:
I grepped that lib for 3 minutes; there might be better ones or even the standard lib to handle this ini-like file.

Reply:
it works. its quite a [bit of code](https://github.com/go-ini/ini/blob/main/parser.go) but a couple of real projects use it. 

I'll take votes for/against using the module from you and @mike-sul. I don't have a strong opinion on it.

Reply:
@doanac it looks like we need to have critical arguments for/against from @mike-sul anyway.

From my side it's easy: this is not the performance critical part of `fioconfig`, thus it is generally a good thing to use a 3PP library which allows to cut our code by 90% (read: decrease our maintenance cost by 80%).

Reply:
here's a different idea. My main reason against this was pulling in yet another 3PP lib to make our go.mod grow. That starts to scare people over time.

However, we already pull in a go TOML library. It seems to be able to parse this shell script. So as long as aklite doesn't do something crazy, we can use it and not add to our SBOM.

Reply:
3cc37da0120f2095b7f3e8ef36276b3e5d33978e

Reply:
This is nice.